### ISSUE TITLE
[BUG] Nonfatal warning "WARNING: [root] One entry (misses or) uses an invalid `type` field ('command')."

### ISSUE THREAD

**Issue by ericfitz** [HorlogeSkynet/archey4#120]
**Describe the bug**
Recently, archey on MacOS Monterey 12.6 started reporting the following error:
```
WARNING: [root] One entry (misses or) uses an invalid `type` field ('command').
```

No configuration file is being provided.  Command line is:
```
archey -l retro
```

Archey4 was installed via HomeBrew.
```
> brew info archey4
==> archey4: stable 4.14.0.0 (bottled)
Simple system information tool written in Python
https://github.com/HorlogeSkynet/archey4
Conflicts with:
  archey (because both install `archey` binaries)
/opt/homebrew/Cellar/archey4/4.14.0.0 (910 files, 10.9MB) *
  Poured from bottle on 2022-09-30 at 18:46:49
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/archey4.rb
License: GPL-3.0-or-later
==> Dependencies
Required: python@3.10 ✔
```

Note that the HomeBrew output reference to "archey" (vs. "archey4") is not relevant; "archey" has never been installed on this system:
```
> brew install archey
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).

Error: archey has been disabled because it has an archived upstream repository!

> brew remove archey
Error: No such keg: /opt/homebrew/Cellar/archey
```

**Expected behavior**
No error

**Screenshots**
```
> archey -l retro
WARNING: [root] One entry (misses or) uses an invalid `type` field ('command').
                            User: xxx
                            Hostname: xxx-mbp-m2.local
                            Model: Mac14.7
               ###          Distro: macOS 12.6 arm64
             ####           Kernel: Darwin 21.6.0
             ###            Uptime: 2 days and 2 minutes
     #######    #######     Load Average: 1.07 1.41 2.15
   ######################   Processes: 546
  #####################     Window Manager: Quartz Compositor
  ####################      Desktop Environment: Aqua
  ####################      Shell: /bin/zsh
  #####################     Terminal: iTerm.app 3.4.16 ██ ██ ██ ██ ██ ██ ██
   ######################   Packages: 117
    ####################    Temperature: Not detected
      ################      CPU: 8 x Apple M2
       ####     #####       GPU: Apple M2
                            RAM: 6054 MiB / 24576 MiB
                            Disk: 109.4 GiB / 2.8 TiB
                            LAN IP: fe80::1, fe80::...
                            WAN IP: 47.x.y.z
```

**Environment**
 - Version used (4.X.Y.Z) : 4.14.0.0
 - Method of installation (distribution package, PyPI, Homebrew, sources) : Homebrew
 - Hardware type (laptop, server, Raspberry, hyper-visor) : MacBook Pro (13-inch, M2, 2022)
 - Python version (3.Y.Z) : 3.10.7
 - Operating system and version : MacOS Monterey 12.6
 - Graphical environment name and version : iTerm2
 - Connectivity (off-line, LAN only, Internet access) : Internet access

**Additional context**
n/a

**Custom configuration**
n/a

**Comment by HorlogeSkynet** [comment]
Hi @ericfitz, could you provide your configuration file content (if any) ?
Thank you, bye 👋

**Comment by ericfitz** [comment]
Hi @HorlogeSkynet , thank you.  I do NOT have a configuration file.

Thank you,
Eric

**Comment by ericfitz** [comment]
I was unable to locate a "default" configuration file; it appears the defaults are in source.  However if there is a default config file somewhere, please point me to it and I'll send.

**Comment by HorlogeSkynet** [comment]
Thanks for your fast answer, so I've narrowed down the source of the issue, and indeed, it's a bug not detected by static typing ☹️
I'll fix it soon and release a new version 🙇

**Comment by ericfitz** [comment]
Thank you!

**Comment by HorlogeSkynet** [comment]
Please see #121 ! Closing here as PR is on its way.
Bye 👋

**Comment by HorlogeSkynet** [comment]
Just released as [v4.14.0.1](https://github.com/HorlogeSkynet/archey4/releases/tag/v4.14.0.1) ! On its way to Homebrew, see Homebrew/homebrew-core#112636.

Thanks, bye :wave:
